docs/source/api/v3/servercheck.rst

```diff
@@ -44,20 +44,20 @@ Request Structure
 .. code-block:: http
 	:caption: Request Example with ``name`` query param
 
-   GET /api/4.0/servercheck?name=edge HTTP/1.1
-   Host: trafficops.infra.ciab.test
-   User-Agent: curl/7.47.0
-   Accept: */*
-   Cookie: mojolicious=...
+  GET /api/4.0/servercheck?name=edge HTTP/1.1
```

Comment:
It looks like your latest commit only deleted a single space, didn't replace the space-based indentation with tabs

Reply:
Seriously.. i did tab them